agent/config/pbench-agent-default.cfg

```diff
@@ -29,7 +29,9 @@ rest_endpoint = api/v%(api_version)s
 server_rest_url = http://%(webserver)s/%(rest_endpoint)s
 
 [pbench/tools]
-default-tool-set = sar, iostat, mpstat, pidstat, proc-vmstat, proc-interrupts, turbostat, perf
+light-tool-set = sar, iostat, mpstat, proc-vmstat, turbostat
+default-tool-set = sar, iostat, mpstat, proc-vmstat, proc-interrupts, turbostat
```

Comment:
Do we really want to change the default? (Will that confuse or depress any existing users?)

In any case, maybe match light/heavy with a medium (even if it's the same now as the new default).

Or even set `default-tool-set = %(medium-tool-set)s` which would set up the three collections as baselines and make it really easy to customize the site default by referencing any of the three.

Reply:
Unless there is a reason to the contrary (i.e., if each set is a super-set of the previous set), I recommend defining each set in terms of the previous one, so that we don't inadvertently omit a tool.  (This also makes it more obvious to the reader what is different about each set.)

Reply:
PTAL at the latest commit: it's still a draft because of test failures, but I'll fix those and open it for final review if everything else looks OK.

Reply:
> Do we really want to change the default? (Will that confuse or depress any existing users?)

Yes, that is the main point - see attached issues. We will need to let users know about it though. I added some comments to the PR comment on how to get back the heavy tool set, if required.